aiutils/tftools/images.py

```diff
@@ -0,0 +1,22 @@
+import tensorflow as tf
+
+def resize_images_like(images, images_like, method=0, align_corners=False):
+    """ Image resize helper.
+
+    Resizes images to the same size as images_like.
+
+    Args:
+      images (4d tensor): Input to resize layer
+        Should have shape `[batch, width, height, channels]`
+
+      images_like (4d tensor): defines shape to resize to
+        Should have shape `[?, width_new, height_new, ?]`
+
+      other arguments see tf.image.resize_images
```

Comment:
It would be better to put those other args here otherwise this function isn't saving the developer much effort.